src/js/fluxcontroller.js

```diff
@@ -106,7 +94,10 @@ define(function (require, exports, module) {
         EventEmitter.call(this);
 
         var cores = window.navigator.hardwareConcurrency || 8;
+
         this._actionQueue = new AsyncDependencyQueue(cores);
+        this._initActionNames();
```

Comment:
Wouldn't it be better to merge `initActionNames` and `initActionLocks in `_createActionReceivers` so we loop through the actions once?